projects/WTI-UI/src/app/modules/clarifications/clarifications.module.ts

```diff
@@ -5,12 +5,14 @@ import { SharedModule } from '../shared/shared.module';
 import { ReactiveFormsModule } from '@angular/forms';
 import { NewClarificationComponent } from './components/new-clarification/new-clarification.component';
 import { NewClarificationAlertComponent } from './components/new-clarification-alert/new-clarification-alert.component';
+import { NewClarificationAnnoucementAlertComponent } from './components/new-announcement-clarification-alert/new-announcement-alert.component';
 
 @NgModule({
   declarations: [
     ClarificationsPageComponent,
     NewClarificationComponent,
-    NewClarificationAlertComponent
+    NewClarificationAlertComponent,
+    NewClarificationAnnoucementAlertComponent
```

Comment:
Component name is misspelled.  Also, see comments on subsequent files; I think this component should be named simply `NewAnnouncementAlertComponent` in order to distinguish "Clarifications" from "Announcements".  Again, see subsequent comments for further details.